libcrux-ml-kem/README.md

```diff
@@ -1,10 +1,15 @@
 # ML-KEM
 
+This crate implements all three ML-KEM ([FIPS 203](https://csrc.nist.gov/pubs/fips/203/final)) variants 512, 768, and 1024.
+For each algorithm, it inlcudes a portable Rust implementation, as well as SIMD implementations for Intel AVX2 and AArch64 platforms.
 
+## Verification
```

Comment:
There's a verification section at the bottom. I'm fine pulling it up here, but we shouldn't have both.

Reply:
Moved.